tests/conftest.py

```diff
@@ -1,9 +1,11 @@
 import pytest
 
+
 def pytest_addoption(parser):
     parser.addoption("--config", action="store")
 
-@pytest.fixture(scope='session')
+
+@pytest.fixture(scope="session")
```

Comment:
It was reformatted and is right here below. The PEP standard must want it to not have a space between the function definition

Reply:
Oh... that's weird.  Yeah, we don't want the newline.  The blank line 7 should be deleted.  I'm sorry I misread that!

Reply:
I updated this file to remove the extra line, but after running `black` on the code again, it looks like it reformats to leave the extra line in. Let me know if you'd like me to remove it manually, although I'm not sure if that would update the linter or not

Reply:
Can you please remove the blank line manually for now?  We need to figure out what Black is thinking there... For one thing... why is it doing that for that decorator, but not for other decorators?

Reply:
Naureen... I may be confused here.  Is it adding a blank line AFTER the decorator?  Or before?  Because before would make perfect sense.  When I look at the raw file, it looks fine.

Reply:
It is adding it before the decorator to look like this: 

```
  1 import pytest
  2 
  3 
  4 def pytest_addoption(parser):
  5     parser.addoption("--config", action="store")
  6 
  7 
  8 @pytest.fixture(scope="session")
  9 def config_file(request):
 10     config_value = request.config.option.config
 11     if config_value is None:
 12         pytest.skip()
 13     return config_value
```

Reply:
Line 7 is the added newline that `black` wants us to keep in

Reply:
I'm so sorry for the back and forth.  That is FINE!  I thought it was doing this:
```
  8 @pytest.fixture(scope="session")
  9
 10 def config_file(request):
 11     config_value = request.config.option.config
 12     if config_value is None:
 13         pytest.skip()
 14     return config_value
```

Reply:
No problem! I'll merge it now!